themes/c8ydocs/layouts/shortcodes/c8y-support-portal.html

```diff
@@ -0,0 +1 @@
+{{- "Cumulocity Support Portal" -}}
```

Comment:
```suggestion
{{- "Cumulocity Support Portal" -}}
```
Customer Service Desk